README.md

```diff
@@ -66,63 +66,65 @@ Execute `sbt clean test` or `sbt clean coverage test && sbt coverageReport` for
 
 Coverage report can be seen at `target/scala-2.13/scoverage-report/index.html`
 
+### Build Docker image
+> You don't have to build the docker image yourself, as the latest master is published at
+> https://hub.docker.com/repository/docker/o360/backend
+Execute `sbt docker:publishLocal`
+
 ### Run application
-To run the application locally:
+To run the application locally or as a docker container:
 
 1. Create `drive_service_key.json` file as described in [Setting up Google account](#setting-up-google-account)
 
-2. Set up database and apply [migrations](#migrations)
+2. Set up PostgreSQL database
 
 3. Set up [auth sources](#setting-up-authentication-sources)
 
 4. Set up configuration:
     * either by changing `conf/application.conf` file
     * or by specifying the [environment variables](#environment-variables)
 
-5. Create `user_invited.html` and `user_approved.html` email templates
+5. Create `user_invited.html` and `user_approved.html` [email templates](#creating-email-templates)
+
+6.
+    * To run locally (on your host): `sbt clean run`
+    * To run as a docker container:
+        > Migrations will be applied automatically at application startup
+        >
+        > It is possible to directly replace application.conf file with preconfigured one instead of using 
+        > environment variables. *application.conf* path inside container - `/opt/docker/conf/application.conf`
+
+        ```shell
+        docker run -d --name open360-api --restart=always -p 9000:9000 \
+            -e DATABASE_USER \
+            -e DATABASE_PASSWORD \
+            -e DATABASE_URL \
+            -e APPLICATION_SECRET \
+            -e EXTERNAL_AUTH_SERVER_URL \
+            -e GOOGLE_REDIRECT_URL \
+            -e GOOGLE_CLIENT_ID \
+            -e GOOGLE_CLIENT_SECRET \
+            -e FACEBOOK_REDIRECT_URL \
+            -e FACEBOOK_CLIENT_ID \
+            -e FACEBOOK_CLIENT_SECRET \
+            -e VK_REDIRECT_URL \
+            -e VK_CLIENT_ID \
+            -e VK_CLIENT_SECRET \
+            -e MAIL_HOST \
+            -e MAIL_PORT \
+            -e MAIL_USER \
+            -e MAIL_PASSWORD \
+            -e MAIL_SEND_FROM \
+            -e SCHEDULER_ENABLED=true \
+            -v $(pwd)/drive_service_key.json:/opt/docker/conf/drive_service_key.json \
+            -v $(pwd)/user_approved.html:/opt/docker/templates/user_approved.html \
+            -v $(pwd)/user_invited.html:/opt/docker/templates/user_invited.html \
+            -v ${USER_FILES_PATH}:/opt/docker/uploads
+            open360/api:latest
+        ```
 
-6. Execute `sbt clean run`
-
-### Build Docker image
-Execute `sbt docker:publishLocal`
```

Comment:
I strongly against removing this command form the readme, it may come in handy when creating an image during the development in forks or else.

Reply:
I have just moved it above the container launching instructions, right after the test ones